scanner-function.tf

```
@@ -1,5 +1,5 @@
 resource "aws_lambda_function" "antivirus-scanner" {
-  function_name = "bucket-antivirus-scanner"
+  function_name = "bucket-antivirus-scanner-${var.env}-${var.region}"
```

```suggestion
  function_name = "${var.env}-bucket-antivirus-scanner"
```
is the current convention I think. Regardless, you don't need to provide the region in the name since functions are per-region